src/main/java/com/coveros/selenified/Capabilities.java

```diff
@@ -78,7 +81,7 @@ public Capabilities(Browser browser) throws InvalidBrowserException {
         setDesiredCapabilities();
     }
 
-    private void setDesiredCapabilities() {
+    private void setDesiredCapabilities() throws InvalidProxyException, InvalidHubException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.coveros.selenified.exceptions.InvalidHubException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)